utils/cargo-gbuild/tests/smoke.rs

```diff
@@ -76,17 +76,26 @@ fn test_program_tests() {
     // This is momently only for adapting the environment (nightly)
     // of our CI.
     {
-        let toolchains = Command::new("rustup")
-            .args(["toolchain", "list"])
+        let targets = Command::new("rustup")
+            .args(["target", "list", "--toolchain", "stable"])
```

Comment:
for prevent unnecessary installation, check the installation of `wasm32-unkown-unkown` directly but not the stable toolchain, because stable toolchain may not have the `wasm32-unknown-unknown` target